pkg/instancetype/controller/vm/controller_test.go

```diff
@@ -948,10 +943,8 @@ var _ = Describe("Instance type and Preference VirtualMachine Controller", func(
 			Expect(err).To(Succeed())
 			Expect(vm.Spec.Instancetype).ToNot(BeNil())
 			Expect(vm.Spec.Instancetype.Name).To(Equal(instancetypeObj.Name))
-			Expect(vm.Spec.Instancetype.RevisionName).ToNot(BeEmpty())
```

Comment:
`Expect(vm.Spec.Instancetype.RevisionName).To(BeEmpty())` until removed?

Reply:
`RevisionName` isn't being removed but I can assert that it's empty in this test sure.

Reply:
Ah yes, but you got the point :)